task-1/task1.go

```diff
@@ -0,0 +1,15 @@
+// lets start
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	a := 0.0
+	b := 0.0
+	fmt.Scanf("%f\n%f", &a, &b)

```

Comment:
в данном случае заданные значения не имеют смысла в программе, так как ты сразу считываешь новые из консоли, поэтому будет лучше так: 
```suggestion
	var a, b float32```
	fmt.Scanf("%f\n%f", &a, &b)

```